src/Abstractions/RetryPolicy.cs

```diff
@@ -123,11 +129,60 @@ public class RetryPolicy
     /// </value>
     public TimeSpan RetryTimeout { get; }
 
+    /// <summary>
+    /// Gets a delegate to call on exception to determine if retries should proceed.
```

Comment:
It may be useful to expand this comment, directing users to `HandleFailure`.

```suggestion
    /// Gets a delegate to call on exception to determine if retries should proceed.
   /// For internal usage. Use <see cref="HandleFailure" /> for setting this delegate.
```